src/components/Facility/LocationManagement.tsx

```diff
@@ -1,9 +1,12 @@
+import { useQuery } from "@tanstack/react-query";
 import { useState } from "react";
 
 import RecordMeta from "@/CAREUI/display/RecordMeta";
 import CareIcon from "@/CAREUI/icons/CareIcon";
 import PaginatedList from "@/CAREUI/misc/PaginatedList";
 
+import { Badge } from "@/components/ui/badge";
+
 import ButtonV2, { Cancel } from "@/components/Common/ButtonV2";
```

Comment:
As mentioned earlier, Button here as well 👍

Reply:
its shadcn badge. do you wish to make it from shadcn button?

Reply:
Yes, there's a button component in components/ui/

Reply:
@Jacobjeevan  buttonV2 has lot more attributes than just className. like href, tooltip, authorizefor, many others. we need to modify the shadcn button. how about i create a new issue to migrate from buttonV2 to shadcn button?

Reply:
> @Jacobjeevan buttonV2 has lot more attributes than just className. like href, tooltip, authorizefor, many others. we need to modify the shadcn button. how about i create a new issue to migrate from buttonV2 to shadcn button?

No, don't modify Shadcn components. You can use AuthorizedButton instead, for example in this [PR](https://github.com/ohcnetwork/care_fe/pull/9377/files#diff-5196ad947a67663db5d6c94e15aaec530c9ac0d0c67bb7dde0a50d774a1ecafc); use corresponding logic with shadcn button (`Link` from raviger for hrefs).